gc/gc-base/src/main/java/org/projectnessie/gc/base/IdentifiedResult.java

```diff
@@ -34,10 +33,10 @@
   // ContentValues per reference per content id.
   private final Map<String, Map<String, ContentValues>> contentValues = new ConcurrentHashMap<>();
 
-  public void addContent(String refName, Content content) {
+  public void addContent(String refName, String contentId, String content) {
```

Comment:
shouldn't the type still stay `Content` rather than using the serialized JSON string?

Reply:
I don't want to deserialize it here as these output will be written to spark rows (#3421) and it needs a serialized string. 
I will add a comment about it.